turbinia/task_manager.py

```diff
@@ -344,6 +343,7 @@ def add_task(self, task, job, evidence_):
       job.tasks.append(task)
     self.state_manager.write_new_task(task)
     self.enqueue_task(task, evidence_)
+    SERVER_TASKS.inc()
```

Comment:
Moving this out of the loop (for task in job_instance.create_tasks([evidence_]):) still ups the counter correctly for each task added?

Reply:
Yes, this will be more accurate since there are other code-paths that can create tasks (e.g. there are "finalize" tasks that  can run after all Tasks in Job complete and after all Jobs/Tasks for a request have been completed).  This is right before the tasks get queued, so it should cover every code path.